pkgs/top-level/all-packages.nix

```diff
@@ -14585,6 +14585,11 @@ in
 
   bastet = callPackage ../games/bastet {};
 
+  beancount = callPackage ../applications/office/beancount {
+      python = python33;
+      pythonPackages = python33Packages;
```

Comment:
according to the docs it needs python 3.3 or above. Therefore, please use python3Packages instead (which currently points to python34Packages). We don't want to have more versions lingering around than really needed.